javascript/javascript3/week3/shoppingCart.js

```diff
@@ -0,0 +1,107 @@
+
+
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  /////The currency convertor not available yet. 

```

Comment:
don't leave commented blocks in the code, they exist in history anyway, and nobody needs to see them besides you